com.chartboost.helium/Android~/helium-android-bridge/src/main/kotlin/com/chartboost/heliumsdk/unity/HeliumUnityAdWrapper.kt

```diff
@@ -0,0 +1,236 @@
+package com.chartboost.heliumsdk.unity
+
+import android.app.Activity
+import android.graphics.Color
+import android.util.DisplayMetrics
+import android.util.Log
+import android.view.Gravity
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.RelativeLayout
+import com.chartboost.heliumsdk.ad.HeliumAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd.HeliumBannerSize
+import com.chartboost.heliumsdk.ad.HeliumFullscreenAd
+import com.chartboost.heliumsdk.ad.HeliumRewardedAd
+import com.unity3d.player.UnityPlayer
+
+class HeliumUnityAdWrapper(private val _ad: HeliumAd?) {
+    // This is the container for the banner.  We need a relative layout to position the banner in one of the 7
+    // possible positions.  HeliumBannerAd is just a FrameLayout.
+    private var mBannerLayout: RelativeLayout? = null
```

Comment:
We also try to not use the `m` prefix in Kotlin.

Reply:
thanks, this is from very legacy code, so I have not had the chance to modify it but I will update it now.